AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -91,6 +91,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 ||cdn.privacy-mgmt.com^$important,domain=promipool.de|android-hilfe.de|wsj.com
 !
 !
+huawei.com##.support-bootom-cookie
```

Comment:
It's ok, but for future pull requests, don't add such rules in the first place. For example, add them a little lower.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/108587567-5f23b980-7365-11eb-8d0e-1cf0c7f819a2.png)

</details><br/>

Reply:
Why though? What conflicts?

Reply:
Our team have added rules to this place as new ones, so when contributors added them at the same place, we have conflicts.
That's why contributors don't have to add rules at the first place.